src/drive/web/modules/viewer/FilesViewer.jsx

```diff
@@ -136,6 +139,14 @@ class FilesViewer extends Component {
             onChangeRequest={this.onChange}
             onCloseRequest={this.onClose}
             renderFallbackExtraContent={file => <Fallback file={file} t={t} />}
+            panelInfoProps={{
+              showPanel: ({ currentFile }) => isInfoPanel(currentFile),
+              PanelContent: ({ currentFile }) => (
+                <WithDriveLocales t={t}>
+                  <PanelContent file={currentFile} />
+                </WithDriveLocales>
```

Comment:
I don't understand why we need to WithDriveLocales wrapper ? When do we lose the context of drive ?

Reply:
Du fait qu'on utilise withLocales dans le Viewer ? https://github.com/cozy/cozy-ui/blob/master/react/Viewer/index.jsx#L206

Reply:
Ah oui OK. Du coup ca rend l'utilisation un peu difficile, on peut pas mettre le withLocales du viewer juste sur els composants qui en ont besoin ? Pour éviter à l'app de devoir faire ça ?

Reply:
Oui bonne idée : https://github.com/cozy/cozy-ui/pull/1717